urllib3/backends/__init__.py

```diff
@@ -1,6 +1,9 @@
-class LoopAbort(Exception):
-    pass
-
-from .trio_backend import TrioBackend
+from urllib3.packages import six
 from .sync_backend import SyncBackend
-from .twisted_backend import TwistedBackend
+
+__all__ = [SyncBackend]
+
+if six.PY3:
```

Comment:
We need a better way to figure out if we're bleached or not... among other things, we may want to run the bleached code on python 3 for testing and other reasons. Maybe we can write `_BLEACHED = False`, and then have the bleaching script detect that specific set of tokens and rewrite it to `_BLEACHED = True`? Anyway, that's just a note for later...

Reply:
Agreed, that would be much better. Leaving that for a future pull request if you do not mind.